test/classes.coffee

```diff
@@ -471,13 +471,6 @@ test "`new` shouldn't add extra parens", ->
   ok new Date().constructor is Date
 
 
-test "`new` works against bare function", ->
-
-  eq Date, new ->
-    eq this, new => this
```

Comment:
We could choose to remove this line only, and continue to test `new -> ...`.

Optionally we could also add a test that `new => ...` fails.